assets/src/hooks/use-copy-ticket.js

```diff
@@ -14,31 +14,34 @@ import useCreateRelationsForTicketToEventDates
 import useCreateRelationsForTicketToPrices
 	from './use-create-relations-for-ticket-to-prices';
 
+const PRICE = 'price';
+const PRICES = 'prices';
+const TICKET = 'ticket';
 const falseFunc = () => false;
 
 const useCopyTicket = ( ticketEntity, dateEntities ) => {
 	const { createEntity } = useDispatch( 'eventespresso/core' );
 	const relatedPrices = useSelect( ( select ) => {
 		const { getRelatedEntities } = select( 'eventespresso/core' );
-		return getRelatedEntities( ticketEntity, 'prices' );
+		return getRelatedEntities( ticketEntity, PRICES );
 	}, [ ticketEntity ] );
-	const newPrices = useCloneEntities( relatedPrices, 'price' );
+	const newPrices = useCloneEntities( relatedPrices, PRICE );
 	const updateTicketDateRelations = useCreateRelationsForTicketToEventDates();
 	const updateTicketPriceRelations = useCreateRelationsForTicketToPrices();
 	return useCallback( async () => {
```

Comment:
here we are already returning a function, no need to return the other one from below